lib/web_ui/test/html/shaders/shader_mask_golden_test.dart

```diff
@@ -19,7 +19,7 @@ const bool debugTest = false;
 
 DomElement get sceneHost =>
     EnginePlatformDispatcher.instance.implicitView!.dom.renderingHost
-        .querySelector('flt-scene-host')!;
+        .querySelector(kSceneHostTagName)!;
```

Comment:
(Note: If you notice we do `DomManager.someElement.querySelector('selector')` in production code, we probably need to cache the element in the `DomManager` as well, so we don't repeat selectors (even though the browser is probably optimizing the selector for us, so this might be a moot point))

Reply:
Are you saying that we do **today** have usages of `DomManager.someElement.querySelector('selector')` in production code, and we should cache them?

Or are you saying **in the future** if we find usages of `DomManager.someElement.querySelector('selector')` in production code, then we should cache them?

(I couldn't find any production code usages in our code, so I'm going to assume you mean the latter).

Reply:
> Or are you saying in the future if we find usages of DomManager.someElement.querySelector('selector') in production code, then we should cache them?

☝️ This is what I meant. Thanks for checking there were not weird usages already!!